server/models/User.js

```diff
@@ -1,6 +1,28 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
+const RatingSchema = new Schema({
+    rating: {
+      type: Number,
+      min: 1,
+      max: 5,
+      required: true,
+      validate: (rating) => {
+        return typeof rating === "number";
+      },
+    },
+    text: {
+      type: String,
+      required: true,
+    },
+    tutor: {
+      type: mongoose.Schema.Types.ObjectID,
+      ref: "User",
+    },
+  }, {
+    timestamps: true
+  });
+
 const UserSchema = new Schema(
   {
     userName: {
```

Comment:
should this have a `type:String` as well ? It looks like the userName object has the profileImage object inside of it

Reply:
@FunkyColdMedyna can you elaborate more what you mean by `userName` having the `profileImage` object? I show `profileImage` as its own separate entity inside the `UserSchema`